Description: Disable compiler check.
 This is not needed in a Debian build environment, and more importantly,
 it breaks cross-builds.
Origin: vendor
Bug-Debian: https://bugs.debian.org/950297
Forwarded: not-needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2020-02-04

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -3,14 +3,6 @@
 use warnings;
 use ExtUtils::MakeMaker;
 
-require "./inc/ExtUtils/HasCompiler.pm";
-
-my $can_xs = ExtUtils::HasCompiler::can_compile_loadable_object( quiet => 1 ) ? 1 : 0;
-if ( !$can_xs ) {
-    print "Sorry! YAML::Syck requires a compiler in order to be built.\n";
-    exit 1;
-}
-
 my $bad;
 if ( eval { require YAML; $YAML::VERSION < 0.60 } ) {
     print "*** Pre-0.60 version of YAML.pm ($YAML::VERSION) detected.\n";
